Intersect.Client/Interface/Game/AdminWindow.cs

```diff
@@ -83,220 +85,144 @@ partial class AdminWindow
         public ImagePanel SpritePanel;
 
         //Init
-        public AdminWindow(Canvas gameCanvas)
+        public AdminWindow(Base gameCanvas)
         {
             mAdminWindow = new WindowControl(gameCanvas, Strings.Admin.title, false, nameof(AdminWindow));
-            mAdminWindow.SetSize(200, 540);
-            mAdminWindow.SetPosition(
-                Graphics.Renderer.GetScreenWidth() / 2 - mAdminWindow.Width / 2,
-                Graphics.Renderer.GetScreenHeight() / 2 - mAdminWindow.Height / 2
-            );
-
+            mAdminWindow.SetPosition(Graphics.Renderer.GetScreenWidth() / 2 - mAdminWindow.Width / 2,
+                Graphics.Renderer.GetScreenHeight() / 2 - mAdminWindow.Height / 2);
```

Comment:
Subtract, then divide. Would be nice if the types were vectors so you could just do `(a - b) / 2` though, but because of GetScreen*() I imagine that's not possible.

Reply:
```csharp
            var windowSize = new Vector2(mAdminWindow.Width, mAdminWindow.Height);
            var screenSize = new Vector2(Graphics.Renderer.ScreenWidth, Graphics.Renderer.ScreenHeight);
            var position = (screenSize - windowSize) / 2;
            mAdminWindow.SetPosition((int)position.X, (int)position.Y);
```

Reply:
I should have been more clear, if Graphics.Renderer had a property called `ScreenDimensions` that returned an int "vector" then we could do it. Don't convert to vectors and then back to an int.

Reply:
```csharp
            mAdminWindow.SetPosition(
                (Graphics.Renderer.ScreenWidth - mAdminWindow.Width) / 2,
                (Graphics.Renderer.ScreenHeight - mAdminWindow.Height) / 2
            );
```